automation/swift-components-docs/generate-swift-project.sh

```diff
@@ -3,8 +3,6 @@
 # This script builds the Rust crate in its directory and generates Swift bindings,
 # headers, and a module map using UniFFI in library mode.
 
-set -euo pipefail  # Ensure script exits on errors or unset variables
-
```

Comment:
1) Why remove this one?

2) This is slightly separate, but can you change the megazord below from `megazord` to `megazord_ios`?  That's the megazord we use on iOS.  Nowadays, I think it contains the same components as the Android megazord, but this will probably change at some point.

Reply:
Fixed both 👍